app/services/idv/profile_maker.rb

```diff
@@ -19,8 +18,8 @@ def save_profile(
       profile.initiating_service_provider = initiating_service_provider
       profile.encrypt_pii(pii_attributes, user_password)
       profile.proofing_components = current_proofing_components
+      profile.verified_at = Time.zone.now unless deactivation_reason || gpo_verification_needed
```

Comment:
👀 gpo_verification_needed was an activation condition https://github.com/18F/identity-idp/pull/8405/files

Reply:
this line is no longer relevant